zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -515,17 +519,18 @@ mod tests {
                         blocks.height IS NULL
                         AND transactions.expiry_height <= blocks_max_height.max_height
```

Comment:
```suggestion
                        AND transactions.expiry_height <= blocks_max_height.max_height
                        AND transactions.expiry_height > 0
```
`put_tx_data` in `wallet.rs` does not map a zero expiry height ([meaning "does not expire"](https://zips.z.cash/zip-0203#specification)) to null, so this would incorrectly include unmined transactions with expiry height 0.